lodash.src.js

```diff
@@ -3592,6 +3594,15 @@
         }
         return fn.apply(thisBinding, args);
       }
+
+      if (isAryExact) {
+        var plist = [];
+        for (var i = 0; i < ary; i++) { plist.push('p' + i); }
+        var params = plist.join(',');
+        var fn = wrapper;
+        eval('wrapper = function(' + params + ') { return fn.apply(this, arguments); }');
```

Comment:
@jdalton moved it into `_.ary` in 30b3083. I should have done that to begin with. I was trying to get a grasp on the code base and chose the hard route. ;)

As for changing to `Function(..)` I'm not sure I understand what you mean. The function body needs to reference values in local scope, so if you're suggesting something like `new Function(param1, param2, body)`, I don't understand how to make that work.